ktor-server/ktor-server-plugins/ktor-server-auth/jvmAndNix/src/io/ktor/server/auth/OAuthProcedure.kt

```diff
@@ -69,17 +69,20 @@ public fun AuthenticationConfig.oauth(
     register(provider)
 }
 
+public class OAuth2RedirectError(public val error: String, public val errorDescription: String?)
+    : AuthenticationFailedCause.Error(if (errorDescription == null) error else "$error: $errorDescription")
+
```

Comment:
Please add documentation for this class.

Reply:
done